.github/workflows/run_shark_tank.yml

```diff
@@ -4,6 +4,7 @@ on:
   schedule:
     - cron: '0 10 * * *'
   workflow_dispatch:
+  pull_request:
```

Comment:
Reminder of removing this before submit.

Reply:
Removed